### PR TITLE
Added json output to ner.py for all entities

### PR DESCRIPTION
**Pull request by beduffy**
Changed generator to list. Added try catch for the odd failure of text file being too big

### CODE REVIEW

File: ner.py [hunk 1]
```diff
@@ -75,13 +80,30 @@ def main():
...
+                print("Processing:", fhandle.name)
+                doc = fhandle.read()
+                ents = extract_entities(doc, entity_types)
+                print("{}: {}".format(fhandle.name, ents))
+                entities_for_file[fhandle.name] = ents
+            except Exception as e:
+                print('Error reading file: \n {}'.format(e))
 
+        if idx % 100 == 0:
+            print('{}/{}')
```
Comment (lgalke) [inline ner.py]: guess the values for the format string are missing

File: ner.py [hunk 2]
```diff
@@ -75,13 +80,30 @@ def main():
...
-            doc = fhandle.read()
-            ents = extract_entities(doc, entity_types)
-            print("{}: {}".format(fhandle.name, ents))
+            try:
+                print("Processing:", fhandle.name)
+                doc = fhandle.read()
+                ents = extract_entities(doc, entity_types)
+                print("{}: {}".format(fhandle.name, ents))
+                entities_for_file[fhandle.name] = ents
+            except Exception as e:
```
Comment (lgalke) [inline ner.py]: Catching everything could make it hard to trace down coding mistakes in the future. What about IOError instead?

File: ner.py (hunk 1 above)
Comment (beduffy) [reply ner.py]: Fixed, noticed it yesterday hahaha.

File: ner.py (hunk 2 above)
Comment (beduffy) [reply ner.py]: Processed 50k papers, only 8 were too big for spacy I assume. 
This was so we wouldn't process those 8 and the whole program wouldn't crash. Was a ValueError though so I'll change it to that

**Pull request merged**